components/errors/response.tsx

```diff
@@ -0,0 +1,32 @@
+interface ResponseCodes {
+  [key: string]: string;
+}
+
+const ErrorResponseCodes: ResponseCodes = {
+  UsernameExistsException: 'An account with the given email already exists.',
```

Comment:
Think it would be better just to have a generic error rather than trying to match on possible exceptions.

Reply:
That's in place now.